packages/react-components/src/templates/SharedResearchOutput.tsx

```diff
@@ -2,12 +2,14 @@ import { ComponentProps } from 'react';
 import { css } from '@emotion/react';
 import { ResearchOutputResponse } from '@asap-hub/model';
 
-import { Card, Headline2, Divider } from '../atoms';
-import { perRem } from '../pixels';
+import { Card, Headline2, Divider, Anchor } from '../atoms';
```

Comment:
Why are you using Anchor instead of ExternalLink?

Reply:
The expected component and the `ExternalLink` component look quite different, I wasn't sure if changing the `ExternalLink` component to support the styles needed for this one was the best approach

Reply:
fair enough, have a look at https://github.com/yldio/asap-hub/pull/970
On Storybook, the component External Link is different from the specs, it matches the External Link on Card spec